lib/salus/bugsnag.rb

```diff
@@ -8,6 +8,11 @@
     notify_endpoint = ENV.fetch('BUGSNAG_ENDPOINT', 'https://notify.bugsnag.com')
     session_endpoint = nil # there are no sessions to track here
 
+    # Bugsnag requires a valid scheme, so add one if it's missing
+    if !notify_endpoint.start_with?('http://', 'https://')
+      notify_endpoint = 'https://' + notify_endpoint
```

Comment:
@cbtoni   Would it make sense to check if notify_endpoint is actually a valid bugsnag URL?  I'm not sure if this is necessary.

Reply:
We could test for a valid url in general, but we have no way to know if it's valid for Bugsnag specifically. I'm assuming this is a rare event and someone would notice after a failed build or whatever